pkg/nfd-master/nfd-master.go

```diff
@@ -162,6 +169,9 @@ func NewNfdMaster(args *Args) (NfdMaster, error) {
 	if args.ConfigFile != "" {
 		nfd.configFilePath = filepath.Clean(args.ConfigFile)
 	}
+	// create the workqueue
+	nfd.queue = workqueue.NewDelayingQueue()
```

Comment:
why use `delaying` queue when you are not using `AddAfter`? It would be better to use standatd queue:
`workqueue.New()` or rate limited one and make it only retry key/node some specified amount of times as in example I have pasted earlier(to prevent infinite processing of one failing key).

Reply:
Good point @PiotrProkop 👍 We don't want to infinitely retry, e.g. nodes that are deleted from the cluster

Reply:
Yes makes sense thanks, do you suggest a max number of retries? @PiotrProkop @marquiz